assets/js/components/notifications/IdeaHubPromptBannerNotification.js

```diff
@@ -35,7 +40,17 @@ import { MODULES_IDEA_HUB } from '../../modules/idea-hub/datastore/constants';
 import { VIEW_CONTEXT_DASHBOARD } from '../../googlesitekit/constants';
 import { trackEvent } from '../../util';
 import BannerNotification from './BannerNotification';
+import PreviewBlock from '../PreviewBlock';
 const { useSelect, useDispatch } = Data;
+const LazyIdeaHubPromptSVG = lazy( () =>
+	import( '../../../svg/graphics/idea-hub-prompt.svg' )
+);
+
+const IdeaHubPromptSVG = ( props ) => (
+	<Suspense fallback={ <PreviewBlock width="100%" height="39.77%" /> }>
+		<LazyIdeaHubPromptSVG { ...props } />
+	</Suspense>
+);
```

Comment:
Let's extract this to its own module for now to avoid reintroducing the added file size to bundles if the svg file were imported directly again. Then we can simply import the `IdeaHubPromptSVG` here which would use the lazy implementation internally.

In the future it would be nice to extract this to a more general abstraction but we don't need to go that far just yet.

Reply:
👍